src/main/java/com/coherentsolutions/pot/insurance/mapper/ClaimMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.coherentsolutions.pot.insurance.mapper;
+
+import com.coherentsolutions.pot.insurance.dto.ClaimDTO;
+import com.coherentsolutions.pot.insurance.entity.ClaimEntity;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface ClaimMapper {
+
+  ClaimMapper INSTANCE = Mappers.getMapper(ClaimMapper.class);
+
+  ClaimDTO claimToClaimDTO(ClaimEntity claim);
```

Comment:
A little bit confused, 
```suggestion
  ClaimDTO entityToDto(ClaimEntity claim);
```

Reply:
Done, changed the naming